config/v1/types_infrastructure.go

```diff
@@ -707,7 +707,6 @@ type AlibabaCloudResourceTag struct {
 // This only includes fields that can be modified in the cluster.
 type NutanixPlatformSpec struct {
 	// prismCentral holds the endpoint address and port to access the Nutanix Prism Central.
-	// When a cluster-wide proxy is installed, this endpoint will not be accessed via the proxy.
```

Comment:
I would rather we update the comment than remove it

```suggestion
// When a cluster-wide proxy is installed, by default, this endpoint will be accessed via the proxy.
// Should you wish for communication with this endpoint not to be proxied, please add the endpoint to the
// proxy spec.noProxy list.
// +kubebuilder:validation:Required
```